rule/line.go

```diff
@@ -25,6 +25,12 @@ var lineRulesJSON = []byte(`
 		"pattern": "AKIA[0-9A-Z]{16}",
 		"caption": "Possible AWS Access Key",
 		"description": null
+	},
+	{
```

Comment:
How susceptible would this be to false positives in checking? Could it be feature flaggable?